website/versioned_docs/version-0.79/native-platform-getting-started.md

```diff
@@ -0,0 +1,92 @@
+---
+id: version-0.79-native-platform-getting-started
+title: Native Platform: Getting Started
+sidebar_label: Getting Started
+original_id: native-platform-getting-started
+---
+
+![Architecture](https://img.shields.io/badge/architecture-new_&_old-green)
+
+Similar to how the [Getting Started for Windows](getting-started.md) guide takes you through the process of creating a base React Native *app* (which supports iOS and Android), and then adding Windows support, this guide will take you through the steps of creating a base React Native *library*, and then adding Windows support.
+
+Before you get started, make sure you have installed all of the [development dependencies](rnw-dependencies.md).
+
+> **Note:** There have always been multiple ways to create base React Native libraries, each creating a slightly different setup. This guide uses the `create-react-native-library` tool (with specific options) to start a brand new library, because that is the specific setup tested by the React Native for Windows team. Your experience may differ when attempting to add Windows support to (existing) libraries created by other tools.
+
+## Create a new React Native library project
+
+Call the following from the place where you want your project directory to live:
+
+<!-- Note, make sure `--react-native-version "XYZ"` are pointing to the correct NPM tags in the command below. -->
+
+<!-- 1. For the next version (i.e. in docs/getting-started.md) use "nightly" for the RN version -->
+<!-- 2. For stable versions in versioned_docs use the semantic version, i.e. "^0.73.0" for the RN version -->
+
+<!-- See https://www.npmjs.com/package/react-native?activeTab=versions for the RN version tags. -->
+
+```bat
+npx --yes create-react-native-library@0.48.9 --react-native-version "nightly" <projectName>
```

Comment:
This page will need to be updated with the new version when snapping the docs. This step should be added to the website bump instructions.

```suggestion
npx --yes create-react-native-library@0.48.9 --react-native-version "^0.79.0" <projectName>
```